packages/core-ui/styles/core.scss

```diff
@@ -270,7 +270,6 @@ SPDX-License-Identifier: Apache-2.0 */
   max-width: 100%;
 
   @media (min-width: (v.$page-width-medium-threshold + 1px)) {
-    padding-right: 0.75rem;
```

Comment:
What happens when this component has a `hOrder` of `0`? It will lose its padding right and there will be no space between it and the next section.